mosaic-site/src/css/global/code.css

```diff
@@ -10,7 +10,7 @@ pre {
   --pre-color: #393a34;
   --code-font-size: 90%;
   --pre-line-height: 1.45;
-  --font-family-monospace: SFMono-Regular, Menlo;
+  --font-family-monospace: var(--salt-text-code-fontFamily);
```

Comment:
this var is only being used in one place. Do you think it makes sense to remove it and use `--salt-text-code-fontFamily` directly?

Reply:
I think this is just to help simply the code on line 72 ` font: var(--code-font-size) / var(--pre-line-height) var(--font-family-monospace);`

Reply:
oh, I thought those vars were created because they had custom values.